game/templates/game/register.html

```
@@ -54,6 +55,32 @@
         <div class="auth-footer">
             Already have an account? <a href="{% url 'login' %}">Sign In</a>
         </div>
+        {% endfor %}
+
+        <button type="submit" class="btn btn-gold">Create Account</button>
+      </form>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </form> ]

(tag-pair)